.github/ISSUE_TEMPLATE/documentation_report.yml

```diff
@@ -86,18 +86,12 @@ body:
   validations:
     required: true
 
-- type: textarea
+- type: markdown
   attributes:
-    label: Additional Information
-    description: |
-      Describe how this improves the documentation, e.g. before/after situation or screenshots.
+    value: |
+      **Additional Information**
 
-      **Tip:** It's not possible to upload the screenshot via this field directly but you can use the last textarea in this form to attach them.
+      Describe how this improves the documentation, e.g. before/after situation or screenshots.
 
       **HINT:** You can paste https://gist.github.com links for larger files.
-    placeholder: >-
-      When the improvement is applied, it makes it more straightforward
-      to understand X.
-  validations:
-    required: true
```

Comment:
Should this be optional on all issue templates?

Reply:
I don't know but it's better to test on one template because I'm not sure how well it'll be combined with WYSIWYG